<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QDialog" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="newsBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>DKU News</string>
   </property>
  </widget>
  <widget class="QPushButton" name="todayBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>DKU Today</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noticeBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>DKU Notice</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
